src/modules/esl-popup/core/esl-popup.ts

```diff
@@ -121,6 +124,21 @@ export class ESLPopup extends ESLToggleable {
     return RTLUtils.isRtl(this) ? 1 - ratio : ratio;
   }
 
+  /** Moves popup into document.body */
+  protected moveToBody(): void {
+    const {parentNode, $placeholder} = this;
+    if (parentNode && parentNode !== document.body) {
+      if ($placeholder) {
+        // to be safe and prevent leaks
+        $placeholder.parentNode?.removeChild($placeholder);
```

Comment:
Minor and only suggestion: Mb we can simplify 
```suggestion
if (!parentNode || parentNode === document.body) return;

// to be safe and prevent leaks
$placeholder && $placeholder.parentNode?.removeChild($placeholder);
```

Reply:
Cool! Thanks!